docs/index.html

```diff
@@ -193,6 +193,15 @@ <h3>Diego Barrios</h3>
                 <a href="https://github.com/diegobahumada" target="_blank"><i class="fab fa-github"></i></a>
             </div>
             </div>        
+        <div class="card">
+            <h3>Gabriel Jofré</h3>
+                <p>Desarrollador autodidacta con agencia activa de desarrollo web/e-commerce Wordpress-headless/Woocommerce junto con el desarrollo de custom snippet y plugin para clientes.</p>
```

Comment:
```suggestion
                <p>Desarrollador autodidacta con agencia activa de desarrollo web/e-commerce Wordpress-headless/Woocommerce junto con el desarrollo de <em>custom snippet</em> y <em>plugin</em> para clientes.</p>
```

Creo que e-commerce es ya una palabra bastante españolizada pero de todas formas se podría poner el tag em